node/lib/util/git_util.js

```diff
@@ -453,6 +453,50 @@ exports.fetch = co.wrap(function *(repo, remoteName) {
     }
 });
 
+/**
+ * Fetch the remote having the specified the specified `repo`.
+ * Throw a `UserError` object if the repository cannot be fetched.
+ *
+ * @async
+ * @param {NodeGit.Repository} repo
+ */
+exports.getRefs = co.wrap(function *(repo) {
+
```

Comment:
What's wrong with http://www.nodegit.org/api/reference/#list ? I don't think we should be using ls-remote anyway -- users should usually have the meta on the same machine as the subrepos.

Reply:
Spoke with @bpeabody about this, he mentioned I should shell out for this. Otherwise, I will have problems with Kerberos.

Reply:
Oh, right -- I forgot that we were using ls-remote.  But we don't want to use ls-remote -- we want to assume that all repos are on the same machine.